addons/onboarding/views/onboarding_views.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="onboarding_view_tree" model="ir.ui.view">
```

Comment:
Guidelines require the full model name :) (``onboarding_onboarding_view_tree``) (and same for other views)